README.md

```diff
@@ -15,8 +15,10 @@ Most income information from the following forms are supported:
 
 So far this project can attach the following schedules to form 1040:
 
+* Schedule 1
```

Comment:
Might want to add a qualifier that the only supported section of schedule 1 is real estate income

Reply:
:heavy_check_mark: